terraform/modules/networking/firewall.tf

```diff
@@ -180,6 +181,58 @@ resource "aws_networkfirewall_rule_group" "stateless_main" {
             }
           }
         }
+
+        # Allow SSH from the GitHub Runner so it can clone from GitHub
+        # Outbound
+        stateless_rule {
+          priority = 4
+          rule_definition {
+            actions = ["aws:pass"]
+            match_attributes {
+              source {
+                address_definition = local.github_runner_cidr_10
+              }
+              source_port {
+                from_port = 1024
+                to_port   = 65535
+              }
+              destination {
+                address_definition = "0.0.0.0/0"
+              }
+              destination_port {
+                from_port = 22
+                to_port   = 22
+              }
+              protocols = [local.tcp_protocol_number]
+            }
+          }
+        }
+
+        # Inbound
```

Comment:
You really need to enable inbound ssh to the github runner?

Reply:
Just tried it and the checkout does indeed fail without the inbound rule